manifests/1.2.0/opensearch-1.2.0.yml

```diff
@@ -64,15 +64,23 @@ components:
     platforms:
       - darwin
       - linux
-  - name: anomaly-detection
```

Comment:
I don't think you meant to remove AD and CCR

Reply:
yes; seem to be an accident, we should not remove other plugins that we don't own from here.

Reply:
accidentally removed during rebase, updated